block/manager.go

```diff
@@ -234,21 +240,28 @@ func (m *Manager) AggregationLoop(ctx context.Context, lazy bool) {
 		time.Sleep(delay)
 	}
 
-	//var timer *time.Timer
-	timer := time.NewTimer(0)
+	blockTimer := time.NewTimer(0)
+	DATimer := time.NewTicker(m.conf.DABlockTime)
```

Comment:
We can use a `time.Ticker` to avoid doing resets.

Reply:
well ones a ticker and ones a timer but the DATimer name doesn't match the struct